### PR TITLE
Improve Dashboard UI

### PR DESCRIPTION
**Pull request by 0mega**
- [x] Moves Dashboard to home page and moves home page contents to wiki
- [x] Improves ticket summary style for cases when words don't fit in one line
- [x] Makes search text optional
- [x] Fix 'result' text to reflect user's search criteria
- [x] Improve search input placeholder
- [x] Show deploys for current day by default

### CODE REVIEW

**Comment by 0mega** [comment]
ping @FundingCircle/black-bat

**Comment by andistuder** [comment]
LGTM

**Comment by dideler** [comment]
When I spoke with @steveweet he mentioned showing deploys for current day as default (without user query). Is that change going to be part of this PR?

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -1,85 +1,39 @@
...
-  A deployer can
-  =link_to "view pending and past deploys to production", releases_url
-  for a specific app. We call these "releases".
-%p
-  A release should have a review associated with it. A link to the review page
-  along with its current status appears in the right-most column.
-%p
-  A release will be flagged as red if it does not have a review, or it does but the work has not been
-  approved (i.e. status is not "Done").
+- if @query || @from_date || @to_date
```
Comment (dideler) [inline app/views/home/index.html.haml]: When I search without any input, all results are returned (limited to first 100) due to these being empty strings. We should tighten this check to make sure they are not empty.

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -3,18 +3,15 @@ class HomeController < ApplicationController
...
     @query = params[:q]
     @from_date = params[:from]
     @to_date = params[:to]
 
-    @tickets = if @query
-                 released_ticket_repo.tickets_for_query(build_query_hash(@query, params))
-               else
-                 []
-               end
+    redirect_to root_path(from: Time.zone.today, to: Time.zone.today) if @query.blank? &&
```
Comment (dideler) [inline app/controllers/home_controller.rb]: **Not a blocker:** can extract the condition to a private method, e.g. `blank_search?`

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -3,18 +3,15 @@ class HomeController < ApplicationController
...
+                                                                         @from_date.blank? &&
+                                                                         @to_date.blank?
 
-    render 'dashboard'
+    @tickets = released_ticket_repo.tickets_for_query(build_query_hash(@query, params))
```
Comment (dideler) [inline app/controllers/home_controller.rb]: **Not a blocker:** Can just pass params and extract query from it

**Comment by dideler** [comment]
LGTM

**Pull request merged**